otx/cli/tools/train.py

```diff
@@ -219,6 +219,7 @@ def train(exit_stack: Optional[ExitStack] = None):  # pylint: disable=too-many-b
     if args.load_weights or args.resume_from:
         ckpt_path = args.resume_from if args.resume_from else args.load_weights
         model_adapters = {
+            "path": ckpt_path,
```

Comment:
It's just a question. can't `weights.pth` be used instead of adding `path` to `model_adapters` newly?

Reply:
args.load_weights or args.resume_from can have not only weights.pth, but also other weight paths, like epoch=0.ckpt or epoch0.pth.
It doesn't matter just for loading weight because we can use weight buffer in model_adapters, but when resuming, checkpoint must be applied in Trainer by using weight path, so adding `path` is needed for this.

Reply:
In mm-familay, that information is saved in "meta". Doesn't it in torch lightning?

Reply:
As far as I understand, lightning uses checkpoint path for resume.
In backend, trainer loads checkpoint and restores configs and metas saved in checkpoint.
Implementing this part seems possible but it may be complicated work and out of this PR scope.
I'll take a look at this point later.